src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.CartDto;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/ecommercee/cart")
+public class CartController {
+    @RequestMapping(method = RequestMethod.GET, value = "getCart")
+    public void getCart(CartDto cartDto) {
```

Comment:
Metoda nie powinna przyjmować argumentu, powinna natomiast zwracać wartość

Reply:
znaczy, żeby stworzyć teraz metodę public List<Cart> getAllCart()  czy na obecną chwilę zrobić @RequestMapping(method = RequestMethod.GET, value = "getCart")
    public void getCart() { return ;}

Reply:
czy w ogóle w wersji:
@RequestMapping(method = RequestMethod.GET, value = "getCart")
public CartDto getCart() { return new CartDto(1L,"first test","description of the first test",8,"1");}

Reply:
Metoda powinna zwracać listę obiektów typu dto